<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryBuilderPackageDialog</class>
 <widget class="QDialog" name="QueryBuilderPackageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пакет запросов</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="btnAdd">
         <property name="toolTip">
          <string>Добавить (Ins)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resurses.qrc">
           <normaloff>:/img/itemAdd.png</normaloff>:/img/itemAdd.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ins</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnAddCopy">
         <property name="toolTip">
          <string>Добавить копированием (F9)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resurses.qrc">
           <normaloff>:/img/copy_24.png</normaloff>:/img/copy_24.png</iconset>
         </property>
         <property name="shortcut">
          <string>F9</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnEdit">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resurses.qrc">
           <normaloff>:/img/itemEdit.png</normaloff>:/img/itemEdit.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnDelete">
         <property name="toolTip">
          <string>Удалить текущий элемент (Del)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resurses.qrc">
           <normaloff>:/img/delete.png</normaloff>:/img/delete.png</iconset>
         </property>
         <property name="shortcut">
          <string>Del</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>2</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::VLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnMoveUp">
         <property name="toolTip">
          <string>Переместить вверх (Ctrl+Shift+Up)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resurses.qrc">
           <normaloff>:/img/moveup.png</normaloff>:/img/moveup.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+Shift+Up</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnMoveDown">
         <property name="toolTip">
          <string>Переместить вниз (Ctrl+Shift+Down)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resurses.qrc">
           <normaloff>:/img/movedown.png</normaloff>:/img/movedown.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+Shift+Down</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="frameShape">
          <enum>QFrame::VLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnOpenDataBase">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resurses.qrc">
           <normaloff>:/img/open_from_disk.png</normaloff>:/img/open_from_disk.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnSaveDatabase">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resurses.qrc">
           <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnQuery">
       <property name="text">
        <string>Запрос</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resurses.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QueryBuilderPackageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QueryBuilderPackageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
